src/app/space/create/deployments/apps/deployments-apps.component.html

```diff
@@ -29,8 +29,23 @@ <h4 class="env-card-title">
   </div>
 
   <div *ngFor="let application of filteredApplicationsList">
-    <deployment-card-container [spaceId]="spaceId | async" [application]="application" [environments]="environments"></deployment-card-container>
+    <deployment-card-container
+      [spaceId]="spaceId | async"
+      [application]="application"
+      [environments]="environments"
+      [modal]="deleteDeploymentModal"
+    ></deployment-card-container>
   </div>
+
+
+  <div class="modal fade" bsModal #modal="bs-modal" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <delete-deployment-modal #deleteDeploymentModal [host]="modal"></delete-deployment-modal>
+      </div>
+    </div>
+  </div>
+
```

Comment:
I find it a little confusing that this modal is located here in this template, but then a reference to it is passed down through the container and into each card. Is this required to have the modal appear as expected within the entire view?